Tests/BuildTests/BuildPlanTests.swift

```diff
@@ -1412,6 +1412,7 @@ final class BuildPlanTests: XCTestCase {
                             providers: [
                                 .brew(["BTarget"]),
                                 .apt(["BTarget"]),
+                                .yum(["BTarget"]),
                             ]
                         )
                     ]),
```

Comment:
Hmm, not sure what's up with the commented out assertion. Can you check?

Reply:
I'll open a separate PR for that, as it's unrelated to this change. If for some reason this has to be reverted, we would regress there again.